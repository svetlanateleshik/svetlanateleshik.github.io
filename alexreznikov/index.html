<!DOCTYPE html>
<html lang="uk-UA">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Приватний виконавець Київ: приватний виконавець Україна</title>
    <meta name="description" content="Приватний державний виконавець Олександр Резніков ✓ Примусове виконання судових рішень в Києві - послуги ★ консультація ★ допомога ☎ (098) 588-14-86"/>
    <link rel="stylesheet" href="style/reset.css">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700,900" rel="stylesheet">
    <link rel="stylesheet" href="slick/slick.css">
    <link rel="stylesheet" href="slick/slick-theme.css">
    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
    <link href="https://fonts.googleapis.com/css?family=El+Messiri:400,700&display=swap&subset=cyrillic" rel="stylesheet">
    <script src="https://kit.fontawesome.com/e2377460b8.js"></script>
    <link rel="stylesheet" href="style/styles.css">

</head>
<body>
    <header class="header">
        <div class="header__inner"> 
            <div class="container">
                <div class="header__inner-nav">
                    <ul>
                    <li><a href="ru.html">RU</a></li>    
                    <li>м. Київ, Бізнес-центр, вул. Преображенська 23, 4 пов. оф. 32</li>
                    <li>forworkreznikov@gmai.com</li>
                    
                    </ul>
                    <ul class="sociallink">
                    
                    <li><a href="https://m.facebook.com/reznikov.alexandr"><img src="img/facebook.png" alt=""></a></li>
                    <li><a href="https://instagram.com/alexandr__reznikov?igshid=jdc0g16vrhm"><img src="img/instagram.png" alt=""></a></li>
                    
                    </ul>
                </div>      
                <div class="flex">
                    <div class="logo">
                        
                           <span>
                             <img src="img/Reznikov_logo_uk.png" alt="">
                            </span> 
                    </div>  
                              
                    <div class="navbar">                    
                                <ul class="navigation">
                                    
                                    <li class="navigation-item"><a href="#why">Про нас</a></li>
                                    <li class="navigation-item"><a href="#brown">Повноваження</a></li>
                                    <li class="navigation-item"><a href="blog.html">Блог</a></li>
                                    <li class="navigation-item"><a href="#partner">Партнери</a></li>
                                    <li class="navigation-item"><a href="#footer">Контакти</a></li>
                                             
                                </ul>
                    </div> 
                    <div class="toggle-hamburger toggle-hamburger__check">
                        <span>menu toggle</span>
                    </div> 
                    <ul class="small-menu">
                                        <li class="navigation-item"><a href="#why">Про нас</a></li>
                                    <li class="navigation-item"><a href="#brown">Повноваження</a></li>
                                    <li class="navigation-item"><a href="blog.html">Блог</a></li>
                                    <li class="navigation-item"><a href="#partner">Партнери</a></li>
                                    <li class="navigation-item"><a href="#footer">Контакти</a></li>
                                </ul> 
                                 
                </div>
            </div>       
        </div> 
        <div class="header__tekst">
            <p>Олександр Резніков</p>
            <h1>Приватний виконавець Виконавчого округу м. Києва</h1>
            
            <button><a href="#why">Детальніше</a></button>
        
        </div> 
        <div class="header__botom">
            
            <div class="header__botom-inner"><a href="#white">Переваги</a></div>
            <div class="header__botom-inner">
                <a href="#brown">Повноваження</a></div>
            <div class="header__botom-inner"><a href="blog.html">Блог</a></div>
           
            
                
            
        </div>  
        <div type="button" class="email-bt">
            <a href="https://t.me/forworkreznikov">
    <div class="text-call">
        <i class="fa fa-envelope" aria-hidden="true"></i>
        <span>Написати<br>мені</span>
    </div>
</a>
</div>


    
        

       <div type="button" class="callback-bt">
        <a href="tel: +38 098 588 14 86" rel="nofollow">
    <div class="text-call">
        <i class="fa fa-phone"></i>
        <span>Замовити<br>дзвінок</span>
    </div>
</a>
</div>         
        
    </header>
    
     
    <section id="white" class="white">
        <div class="container__inner">
            <div class="services__wrap-grey">
                <div class="services">                
                    <p class="services__title-grey">Переваги</p>
                    <div class="services__item-grey">
                        <div class="services__item-bl"> 
                        <img src="img/Відкриття виконавчого провадження1.png" alt="">               
                            <p>Відкриття виконавчого провадження в день надходження заяви Стягувача</p>
                        </div>
                        <div class="services__item-bl">
                            <img src="img/Офіційно надані повноваження1.png" alt="">
                            <p>Офіційно надані державою повноваження: право у межах відкритого виконавчого провадження арештовувати майно, стягувати кошти, безперешкодно входити на територію юридичної особи, залучати правоохоронні органи</p>
                        </div>
                        <div class="services__item-bl"> 
                        <img src="img/Повне та постійне інформування1.png" alt="">              
                            <p>Повне та постійне інформування Стягувача щодо усіх заходів примусового виконання</p>
                        </div>
                        <div class="services__item-bl">
                            <img src="img/Збереження таємниці1.png" alt="">
                            <p>Збереження таємниці щодо інформації, яка стала відома під час провадження</p>
                        </div>
                        <div class="services__item-bl"> 
                        <img src="img/Максимальна зацікавленність1.png" alt="">              
                           <p>Максимальна зацікавленість у фактичному виконанні судових рішень на користь Стягувача, адже винагорода приватного виконавця стягується з Боржника, а за Вами залишається право на винагороду роботи виконавця за підсумком роботи</p>
                        </div>
                        <div class="services__item-bl">
                            <img src="img/Відсутність бюрократичних перешкод1.png" alt="">
                            <p>Відсутність бюрократичних перешкод та черг, своєчасне та професійне виконання рішень</p>
                        </div>
                        
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section id="brown" class="brown">
        <div class="container__inner">
            
                <div class="services__wrap-grey">
                     <div class="services">                
                            <p class="services__title-white">Повноваження</p>
                            <div class="services__item-grey">
                                <div class="services__item-b">
                                    <a href="styagnennyakoshtiv.html">
                                    <img src="img/Звернення стягнення на майно та кошти боржника.png" alt="">               
                                   <p>Звернення стягнення на майно та кошти боржника</p>
                                   </a>
                                </div>

                                
                                <div class="services__item-b">
                                    <a href="rozshuk.html">
                                    <img src="img/Розшук майна.png" alt="">
                                    <p>Розшук майна</p>
                                    </a>
                                </div>
                                <div class="services__item-b"> 
                                <a href="vhodjennja.html">
                                  <img src="img/Примусове входження до приміщення.png" alt="">           
                                    <p>Примусове входження до приміщення</p>
                                    </a> 
                                </div>
                                <div class="services__item-b">
                                    <a href="prityagnennya.html">
                                    <img src="img/Притягнення до відповідальності.png" alt="">
                                    <p>Притягнення до відповідальності</p>
                                    </a>
                                </div>
                                <div class="services__item-b"> 
                                <a href="areshtkoshtiv.html">
                                  <img src="img/Арешт коштів боржника.png" alt="styagnennyakoshtiv.html">           
                                   <p>Арешт коштів боржника</p>
                                   </a> 
                                </div>
                                <div class="services__item-b">
                                    <a href="areshtmaina.html">
                                    <img src="img/Арешт майна боржника.png" alt="">
                                    <p>Арешт майна боржника</p>
                                    </a>
                                </div>
                                <div class="services__item-b"> 
                                <a href="obmezhennya.html">
                                  <img src="img/Обмеження боржника у праві виїзду за кордон.png" alt="">           
                                   <p>Обмеження боржника у праві виїзду за кордон</p>
                                   </a> 
                                </div>
                                <div class="services__item-b">
                                    <a href="">
                                    <img src="img/Розшук боржника.png" alt="">                
                                    <p>Розшук боржника</p>
                                </a>
                                </div>
                                
                                
                            </div>
                         
                     </div>
                </div>
               
        </div>
    </section>
    <section>
        
            <div class="numbers">

                <div id="counter" class="counter">
                    <div class="why__wrap">
                        <div class="why__wrap-inner">
                            <span>За 5 кроків до приватного виконавця</span>
                            <div class="block">
                                <img src="img/Вивчав право..белым.png" alt="">
                                <p>Вивчав право у Національному юридичному університеті імені Ярослава Мудрого.</p>
                            </div>
                            <div class="block">
                            <img src="img/Працював у адвокатському...белым.png" alt="">
                                <p>Працював у Адвокатському бюро.</p>
                            </div>
                            <div class="block">
                                <img src="img/Працював у Солом’янському...белым.png" alt="">
                            <p>Працював державним виконавцем Солом'янського районного відділу державної виконавчої служби міста Києва Головного територіального управління юстиції у місті Києві.
                            </p>
                            </div>
                            <div class="block">
                                <img src="img/Працював виконавцем примусового виконання...белым.png" alt="">
                            <p>Працював державним виконавцем примусового виконання рішень державної виконавчої служби Головного територіального управління юстиції у Київській області.</p>
                            </div>
                            <div class="block">
                                <img src="img/Процював помічником...белым.png" alt="">
                                <p>Працював помічником приватного виконавця.</p>
                            </div>            
                        </div>
                    </div>
                </div>
            
        </div>        
    </section> 
    <section id="white" class="white">

         <div class="container__inner">
         <div class="services__wrap-grey">
             <div class="services">                
                    
                    
                 <p class="services__title-grey">Публікації</p>
             </div>
         </div>

         <div class="slider">
            <div class="slider__inner">
                <a href="blog.html">
                    <img src="img/IMG_3982_cr.jpg" alt="">
                    <div class="slider__inner-text">
                        <p>Не таємниця, що введення нової професії приватного виконавця - один з важливіших етапів судової реформи в Україні. 
                        Яскравим підтвердженням цього став підвищений інтерес до міжнародньої конференції "Становлення та функціонування системи приватного виконання судових рішень", котра відбулась сьогодні. поради посприяють продуктивному старту нової діяльності
                        </p>
                    </div>
                </a>
            </div>
            <div class="slider__inner">
                <a href="blog.html">    
                    <img src="img/IMG_3983_cr1.jpg" alt="">
                    <div class="slider__inner-text">
                        <p>30 березня 2018 року Міністерство юстиції України спільно з Асоціацією приватних виконавців України і Київським національним університетом імені Тараса Шевченко,  провели науково-практичну конференцію «Реформа виконавчого провадження: сьогодення та перспективи».Всі обговорювані спікерами теми були важливі і актуальні. Кваліфікація приватних виконавців зростає щодня!
                        </p>
                        <!--p>
                        за підтримки Програми Агентства США з міжнародного розвитку (USAID) «Нове правосуддя», Проекту Європейського Союзу «Підтримка реформ юстиції і правосуддя в Україні (ПРАВО-JUSTICE)» і Центру комерційного права (Україна)/p-->   
                    </div>
                </a>
            </div>
            <div class="slider__inner">
                <a href="blog.html">
                    <img src="img/photo2_cr.jpg" alt="">
                    <div class="slider__inner-text">
                        <p>Реформа о приватних виконавцях в Украині успішно впроваджується в практику і відточується день у день. Зокрема, за рахунок регулярного відпрацювання усунення виникаючих складностей.
                        Чудовим привідом для цього став II-й Міжнародний форум, присвячений темі виконавчого виробництва. Результати виконанної колосальної командної праці - кращий доказ эфективності нововведення</p>
                    </div>
                </a>
            </div>
            <div class="slider__inner">
                <a href="blog.html">
                    <img src="img/photo3_cr.jpg" alt="">
                    <div class="slider__inner-text">
                        <p>Центр коммерційного права разом з Міністерством юстиції провів семінар для приватних виконавців, під час якого обговорювались питання відпрацювання та оптимізації робочого процесу приватниї виконавців. Семінар вели спеціалист з діловодства з Министерства юстиції, эксперт з оподаткування й HR-консультант.
                            <!--p>
                        Всі затронуті спікерами теми були важливі й актуальні. Кваліфикація приватних виконавців росте день у денб</p-->
                    </div>
                </a>
            </div>
         </div>        
    </section>
    <section id="partner" class="white">

         <div class="container__inner">
         <div class="services__wrap-grey">
             <div class="services">                
                    <p class="services__title-grey">Партнери</p>
                    
                    <div class="services__item-grey">
                        <div class="services__item-bl bank">                
                            
                        </div>
                      
                        <div class="services__item-bl bank1">
                            
                        </div>
                        <div class="services__item-bl bank2">
                            
                        </div>
                        <div class="services__item-bl bank3">
                            
                        </div>
                        <div class="services__item-bl bank4">
                            
                        </div>
                        <div class="services__item-bl bank5">
                            
                        </div>
                        <div class="services__item-bl bank6">
                            
                        </div>
                        <div class="services__item-bl bank7">
                            
                        </div>
                        <div class="services__item-bl bank8">
                            
                        </div>
                        <div class="services__item-bl bank9">
                            
                        </div>
                        <div class="services__item-bl bank10">
                            
                        </div>
                       
                        
                        
                            
                        
                        <div class="services__item-bl bank13">
                            
                        </div>
                        
                        
                        

                        
                    </div>
                    
                 
             </div>
         </div>
         </div>        
    </section>
    <footer id="footer" class="footer">
    <div class="container">
        <div class="footer__wrap">
            <div class="footer__contact">
                <strong class="footer__contact-title">Контакти</strong>
                <ul class=contact-list>
                    <li><strong>Телефони:</strong></li>
                    <li><a href="tel:+38(098)-588-14-86">+38(098)-588-14-86</a></li>
                </ul>
                <ul class=contact-list>
                    <li><strong>Адреса:</strong></li>
                    <li>м. Київ, Бізнес-центр, вул. Преображенська 23, 4 пов. оф. 32</li>
                </ul>
                <ul class=contact-list>
                    <li><strong>Години роботи:</strong></li>
                    <li>Пн-Пт: з 09:00 до 18:00</li>
                    <li>Cб-Нд за домовленістю</li>
                </ul>
            </div>
            <div class="footer__iframe">
                <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2541.990061427866!2d30.476930295384243!3d50.422656803304974!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4cec5e01d2641%3A0xe5ffe2a40ae1efc2!2z0YPQuy4g0J_RgNC10L7QsdGA0LDQttC10L3RgdC60LDRjywgMjMsINCa0LjQtdCyLCAwMjAwMA!5e0!3m2!1sru!2sua!4v1572899109004!5m2!1sru!2sua" width="100%" height="300" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
                
                <ul class="navigation-footer">
                     
                     <li><a href="index.html">Про нас</a></li>
                     <li><a href="#brown">Повноваження</a></li>
                    <li><a href="blog.html">Блог</a></li>
                     <li><a href="#partner">Партнери</a></li>
                </ul>
            </div>
            <div class="logorez">
              <img src="img/shield.png" alt="">
            </div>      
      </div>
    </div>
  </footer>



<script src="js/jquery-3.2.1.min.js"></script>
<script src="js/slick.min.js"></script>
<script src="js/scripts.js"></script>
</body>

</html>