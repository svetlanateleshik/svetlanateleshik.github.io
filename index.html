<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>AlexReznikov</title>
    <link rel="stylesheet" href="style/reset.css">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700,900" rel="stylesheet">
    <link rel="stylesheet" href="style/slick.css">
    <link rel="stylesheet" href="style/slick-theme.css">
    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
    <script src="https://kit.fontawesome.com/e2377460b8.js"></script>
    <link rel="stylesheet" href="style/styles.css">

</head>
<body>
    <header class="header">
        <div class="header__inner">
        
            <div class="container flex">
                <div class="logo">
                
                   <span>
                     <img src="img/Reznikov.png" alt="">
                    </span> 
                </div>
            
                <div class="navbar">
                    
                            <ul class="navigation">
                                <li class="navigation-item "><a href="#" class="active">Головна сторінка</a></li>
                                <li class="navigation-item"><a href="#">Про мене</a></li>
                                <li class="navigation-item"><a href="#">Послуги</a></li>
                                <li class="navigation-item"><a href="#">Блог</a></li>
                        <li class="navigation-item"><a href="#">Контакти</a></li>
                            </ul>
                </div>
                <div class="header__contakt-list">
                    <ul>
                        <li>
                            <a href="tel:+38 098 588 14 86
" title="+38 098 588 14 86
">+38 098 588 14 86
</a>
                        </li>
                        <li>
                            <a href="forworkreznikov@gmai.com
" title="forworkreznikov@gmai.com
">forworkreznikov@gmai.com
</a>
                        </li>
                        <li>
                            <a href="https://www.facebook.com/reznikov.alexandr" title="reznikov.alexandr
">reznikov.alexandr<i class=ion-social-facebook></i>
                            </a>
                        </li>
                        
                        <li><a href="">м. Київ, вул. Стрілецька, 7/6</a></li>





                    </ul>
                    
                </div>
            </div>
       
         </div>             
            <section class="slider-section">
            
                 <div class="main-slider">
                    <div>
                        <div class="mainslider">
                        <img src="img/header.jpg">  
                        </div> 
                        <div class="text-center">
                            <ul>
                                <li>

<a href="" title="Примусове виконання судових рішень">
Примусове виконання судових рішень</a>

</li>
                            <li>

<a href="" title="Примусове виконання судових рішень">
Комплексна допомога у вирішенні питань з боргами</a>

</li>
   

</ul>
                            
                            
                            
                            
                        </div>
                    </div>
                    <div>
                        <div class="mainslider">
                        <img src="img/header.jpg"> 
                        </div> 
                        <div class="text-center ">
                            <ul>

                          <li>

<a href="" title="Примусове виконання судових рішень">
Юридичні послуги у кримінальних справах</a>

</li>>
                                <li>

<a href="" title="Юридичні послуги у спадкових справах">
Юридичні послуги у спадкових справах</a>

</li>
                            
   
   

</ul>
                        </div>
                    </div>
                    <div>
                        <div class="mainslider">
                        <img src="img/header.jpg"> 
                        </div> 
                        <div class="text-center">

                            <p>Юридичні послуги</p>
                        </div>
                    </div>           
                 </div>
             </section>
         
    </header>
    <section>
        <div class="white container">
            <div class="text">
                <h2>Обо мне</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rerum nesciunt incidunt illo reiciendis, corporis provident, deserunt eius iusto enim labore tempore repellat! Deleniti veniam quidem labore possimus quas sit laboriosam. Blanditiis consectetur architecto quas dolores laboriosam pariatur incidunt aperiam, labore quam dicta exercitationem suscipit, nemo velit quibusdam debitis modi? Autem, ex impedit est repellendus amet quam minima voluptatem commodi rem eum quisquam accusamus atque possimus perferendis velit! Enim quae rerum, labore odit pariatur debitis quas iusto quis quam natus minus quidem dolore, consequatur laborum neque numquam autem sit dolorum laboriosam eveniet dolores. Esse et optio, quidem blanditiis labore delectus dolores.</p>
            </div>
            <div class="foto">
                <img src="img/man.jpg" alt="">
            </div>
            
        </div>
    </section>
    <section class="brown">
     
        
    </section>
    
    <section></section>
    <section></section>
    <section id="contact">
     
        
      


       
    




</section>



    <section id="services">
        
        
        
    
    </section>
    
    
    <footer id="footer" class="footer">
    <div class="container">
        <div class="footer__wrap">
        <div class="footer__contact">
            <strong class="footer__contact-title">Контакти</strong>
            <ul class=contact-list>
            <li><strong>Телефони:</strong></li>
            <li><a href="tel:+38(098)-588-14-86">+38(098)-588-14-86</a></li>
            </ul>
            <ul class=contact-list>
            <li><strong>Адреса:</strong></li>
            <li>м. Київ, вул. Стрілецька, 7/6</li>
            </ul>
            <ul class=contact-list>
            <li><strong>Години роботи:</strong></li>
            <li>Пн-Пт: з 09:00 до 18:00</li>
            <li>Cб-Нд за домовленістю</li>
            </ul>
        </div>
        <div class="footer__iframe">
            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2540.35943718261!2d30.50987011504767!3d50.45303109506102!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4ce5c77f7e31d%3A0xcbc511399686de2e!2z0YPQuy4g0KHRgtGA0LXQu9C10YbQutCw0Y8sIDcvNiwg0JrQuNC10LIsIDAyMDAw!5e0!3m2!1sru!2sua!4v1564946514327!5m2!1sru!2sua" width="100%" height="300" frameborder="0" style="border:0" allowfullscreen></iframe>
            <ul class="navigation-footer">
         <li><a href="index.html">Головна сторінка</a></li>
         <li><a href="#services">Про мене</a></li>
         <li><a href="#prices">Послуги</a></li>
        <li><a href="#prices">Блог</a></li>
         <li><a href="#contact">Контакти</a></li>
      </ul>
        </div>
        <div class="logorez">
          <img src="img/shield.png" alt="">
      </div>
      
      </div>
    </div>
  </footer>

<script src="js/jquery-3.2.1.min.js"></script>
<script src="js/slick.min.js"></script>
<script src="js/scripts.js"></script>
</body>

</html>