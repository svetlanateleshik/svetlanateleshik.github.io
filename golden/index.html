<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="styles/styles.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
					<nav>
						<ul class="menu">
							<li>
								<a href="#">Home</a>
							</li>
							<li>
								<a href="#">Services</a>
							</li>
							<li><a href="#">Portfolio</a></li>
							<li><a href="#">About</a></li>
							<li><a href="#">Contact</a></li>
						</ul>
					</nav>
					<div class="menu-toogle">
                        <span></span>
                    </div>
                    <ul class="small-menu">
										<li><a class="link" href="index.html">Home</a></li>
										<li><a class="link" href="#servis">services</a></li>
										<li><a class="link" href="#portfolio">portfolio</a></li>
										<li><a class="link" href="#about">about</a></li>					
										<li><a class="link" href="#contact">contact</a></li>
								</ul>
			</div>
		<div class="titles">
			<div class="titles__first">
				<p>Welcome To Our Studio!</p>
				
			</div>
			<h1>
				<p>It’s nice to Meet you</p>
				
			</h1>
			<a class="button" href="#">tell me more</a>


		</div>
		</div>
	</header>

	<section id="services">
		<div class="container">
		<div class="title">
			<h2>
				services
			</h2>
			<p>Proin iaculis purus consequat sem cure. </p>
		</div>
		<div class="services clearfix">
			<div class="services__item">
				<img src="img/icon1.png" alt="">
				<h3>E-Commerce</h3>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
			</div>
			<div class="services__item">
				<img src="img/icon2.png" alt="">
				<h3>Responsive Web</h3>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
			</div>
			<div class="services__item">
				<img src="img/icon3.png" alt="">
				<h3>Web Security</h3>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
			</div>
		</div>
	</div>
	</section>
	<section id="portfolio">
		<div class="container">
		<div class="title">
			<h2>
				Our portfolio
			</h2>
			<p>Proin iaculis purus consequat sem cure. </p>
		</div>
		<div class="works clearfix">
			
				<img src="img/img1.jpg" alt="работа">
				<img src="img/img2.jpg" alt="работа">
				<img src="img/img3.jpg" alt="работа">
		</div>

				
		</div>
	
	</section>
	<section id="about-us">
		<div class="container">
			<div class="title">
				<h2>
					About us
				</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>
			<div class="about__item">
				<div class="item">
				
				<div class="text left">
					<h3>July 2016</h3>
					<span>Our Humble Beginnings</span>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about1.jpg" alt="">
			</div>
			<div class="item">
				<div class="text right">
					<h3>January 2017</h3>
					<span>Facing Startup Battles</span>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about2.jpg" alt="">
			
			</div>
			<div class="item">
				<div class="text left">
					<h3>December 2017</h3>
					<span>Enter The Dark Days</span>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about3.jpg" alt="">
			
			</div>
			<div class="item">
				<div class="text right">
					<h3>February 2018
					</h3>
					<span>Our Triumph</span>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about4.jpg" alt="">
			</div>
									

		</div>
	</section>
	<footer id="footer" class="footer">
    <div class="container">
      <ul class="navigation-footer">
         <li><a href="index.html">home</a></li>
         <li><a href="#services">service</a></li>
         <li><a href="#portfolio">portfolio</a></li>
         <li><a href="#about">about</a></li>
         <li><a href="#contact">contact</a></li>
      </ul>
    </div>
  </footer>
	<script src="js/jquery-3.3.1.min.js"></script>
	<script src="js/scripts.js"></script>
</body>
</html>